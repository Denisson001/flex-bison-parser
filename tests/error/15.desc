Compilation error - invalid bool expression