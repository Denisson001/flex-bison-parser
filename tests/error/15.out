Parser: —Åompilation error - syntax error, unexpected ')', expecting '=' or '!' or '<' or '>' near line 3
